src/helpers.js

```diff
@@ -136,7 +134,7 @@ export function normalizeSwagger(parsedSpec) {
   for (const pathName in paths) {
     const path = paths[pathName];
 
-    if (!isObject(path)) {
+    if (typeof path !== 'object') {
```

Comment:
`isObject` cannot be simply replaced with `typeof path === 'object'`; this is not functionally equivalent.

The source code of `isObject` is as follows:

```js
    function isObject(value) {
      var type = typeof value;
      return value != null && (type == 'object' || type == 'function');
    }
```

Which means that anything that reports typeof `object` || `function` except `null` and `undefined`.